### PR TITLE
Add implementation of JsonSerializable to MongoId

### PR DESCRIPTION
**Pull request by idr0id**

### CODE REVIEW

File: lib/Mongo/MongoId.php [hunk 1]
```diff
@@ -218,4 +218,14 @@ private function createObjectID($id)
             throw new MongoException('Invalid object ID', 19);
         }
     }
+
+    /**
+     * @return object
```
Comment (alcaeus) [inline lib/Mongo/MongoId.php]: Should be `@return stdClass`

File: lib/Mongo/MongoId.php [hunk 2]
```diff
@@ -218,4 +218,14 @@ private function createObjectID($id)
...
+     */
+    function jsonSerialize()
```
Comment (alcaeus) [inline lib/Mongo/MongoId.php]: Missing `public` visibility.

**Comment by alcaeus** [comment]
Thanks for the PR! Can you add tests for it?

**Comment by idr0id** [comment]
Done!

**Comment by idr0id** [comment]
Maybe make PR for backporting to 1.0.x?

**Comment by idr0id** [comment]
@alcaeus ping

**Comment by alcaeus** [comment]
Apologies for the delay, I'm taking some personal time and only sporadically have time to do some coding. I've backported your changes into the 1.0 branch and will merge once the PR is stable. Thanks for the contribution 🍻

**Pull request closed without merging**